### DISCUSSION TITLE
Push notification for realtime firebase data update

### DISCUSSION THREAD

**Discussion by mrnams** [dotnet/maui#20963] [General]
Hi,
Sorry for dumb question.
I have developed android app using MAUI (.net 8.0) to control home appliances, where app read data from google real time Firebase data, and i could do it.
Now, I want push notification as soon as data in firebase get updated?
Please note before posting here, I have googled and did R&D, but could not succeed.
**Can anyone please point me to any example?**

**Comment by davidortinau** [comment]
https://shinylib.net/client/push/

**Reply by mrnams** [reply]
Thanks, let me go through it.

**Reply by mrnams** [reply]
Hello,I could able to get Push notification using FCM, I want these notification when value in realtime database get updated.
